### PR TITLE
Adds explicit web links to the environment.yml and the requirements.txt files

### PR DESCRIPTION
**Pull request by JacekHoleczek**
Adds explicit web links to the `environment.yml` and the `requirements.txt` files (important for users who want to try the tutorial on their own or their lab’s computer).

BTW. Note that the versions of modules in the `environment.yml` file are a bit outdated.

### CODE REVIEW

**Comment by GuillermoFidalgo** [comment]
Requires approval
Anyone?

File: setup.md [hunk 1]
```diff
@@ -71,13 +71,13 @@ Simply click the following button:
...
 
 ```bash
 conda install uproot awkward   # ... others?
 ```
 
-Alternatively, you can install all required packages locally with pip: Take a look at requirements.txt for a list.
+Alternatively, you can install all required packages locally with pip: Take a look at the [environment.yml](https://github.com/hsf-training/hsf-training-scikit-hep-webpage/blob/main/environment.yml) and the [requirements.txt](https://github.com/hsf-training/hsf-training-scikit-hep-webpage/blob/main/requirements.txt) for a list (this includes all dependencies listed under the pip key and some of the dependencies listed above).
```
Comment (wdconinc) [inline setup.md:80]: Saying "all" is not quite correct since I don't think you can install xrootd via pip.

**Review by wdconinc — COMMENTED** [review]
LGTM

**Review by wdconinc — APPROVED** [review]

File: setup.md (hunk 1 above)
Comment (eduardo-rodrigues) [reply setup.md:80]: You can in fact, see https://pypi.org/project/xrootd/. Am going to approve and merge: seems this got forgotten.

**Review by eduardo-rodrigues — APPROVED** [review]

**Pull request merged**